en/operations/admin-procedures.html

```diff
@@ -121,6 +121,217 @@ <h2 id="status-pages">Status pages</h2>
+  also dump current index structures, see <code>artist</code> as an attribute:
+</p>
+<pre>
+$ curl "http://localhost:8080/search/?ranking=rank_albums&yql=select%20%2A%20from%20sources%20%2A%20where%20artist%20contains%20%22Coldplay%22%3B"
+
+$ docker exec vespa /usr/bin/sh -c '/opt/vespa/bin/vespa-proton-cmd --local triggerFlush' && \
+  docker exec vespa /usr/bin/sh -c '/usr/bin/find /opt/vespa/var/db/vespa/search/cluster.music/n0/documents/music/0.ready'
+</pre>
+<p>
+  Add index aspect and match/stemming settings to the field, deploy and observe output
+</p>
+<pre>
+field artist type string {
+  indexing: summary | attribute <span style="background-color: yellow;">| index
+    match    : word
+    stemming : none</span>
+}
+
+$ (cd src/main/application && zip -r - .) |   curl --header Content-Type:application/zip --data-binary @- \
+  localhost:19071/application/v2/tenant/default/prepareandactivate
+
+{
+  "log": [
+    {
+      "time": 1628239290150,
+      "level": "WARNING",
+      "message": "Change(s) between active and new application that may require re-index:\nindexing-change:
+                  Consider re-indexing document type 'music' in cluster 'music' because:\n
+                  1) Document type 'music': Field 'artist' changed: add index aspect,
+                  indexing script: '{ input artist | summary artist | attribute artist; }' ->
+                                   '{ input artist | exact | summary artist | attribute artist | index artist; }'\n"
+    }
+  ],
+  "tenant": "default",
+  "url": "http://localhost:19071/application/v2/tenant/default/application/default/environment/prod/region/default/instance/default",
+  "message": "Session 3 for tenant 'default' prepared and activated.",
+  "configChangeActions": {
+    "restart": [],
+    "refeed": [],
+    "reindex": [
+      {
+        "name": "indexing-change",
+        "documentType": "music",
+        "clusterName": "music",
+        "messages": [
+          "Document type 'music': Field 'artist' changed: add index aspect, indexing script: '{ input artist | summary artist | attribute artist; }' -> '{ input artist | exact | summary artist | attribute artist | index artist; }'"
+        ],
+        "services": [
+          {
+            "serviceName": "searchnode",
+            "serviceType": "searchnode",
+            "configId": "music/search/cluster.music/0",
+            "hostName": "vespa-container"
+          }
+        ]
+      }
+    ]
+  }
+}
+</pre>
+<p>
+  Trigger reindexing, deploy again, wait until done
+  (look for the log message / use the <code>reindexing</code> endpoint).
+  Dumping the index structures now show artist both in index and attribute.
+  Verify the query still works:
+</p>
+<pre>
+$ curl http://localhost:19071/application/v2/tenant/default/application/default/environment/prod/region/default/instance/default/reindex
+
+{
+  "generation": 3,
+  "applicationPackageFileReference": "",
+  "modelVersions": ["7.423.37"]
+}
+
```

Comment:
```suggestion
$ curl http://localhost:19071/application/v2/tenant/default/application/default/environment/prod/region/default/instance/default/reindex

{
  "generation": 3,
  "applicationPackageFileReference": "",
  "modelVersions": ["7.423.37"]
}

```

This seems to match something completely irrelevant? That path is used with POST/DELETE to (de)activate reindexing, AFAIR, but isn't documented.